documentation/modules/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -45,22 +52,33 @@ spec:
   zookeeper:
     # ...
 ----
+<1> Ingress hosts for the bootstrap service and kafka brokers.
 
-. Make sure the hosts in the `configuration` section properly resolve to the Ingress endpoints.
+. Make sure the hosts in the `configuration` resolve to the Ingress endpoints.
++
+[source,shell,subs=+quotes]
```

Comment:
@scholzj - Not sure what this step is meant to do. So added some placeholder commands!